autoware.repos

```diff
@@ -49,6 +49,18 @@ repositories:
     type: git
     url: https://github.com/tier4/pointcloud_to_laserscan.git
     version: tier4/main
+  universe/external/eagleye:
+    type: git
+    url: https://github.com/MapIV/eagleye.git
+    version: autoware-main
+  universe/external/rtklib_ros_bridge:
+    type: git
+    url: https://github.com/MapIV/rtklib_ros_bridge.git
+    version: ros2-v0.1.0
+  universe/external/llh_converter:
+    type: git
+    url: https://github.com/MapIV/llh_converter.git
```

Comment:
I see some packages([gnss_poser](https://github.com/autowarefoundation/autoware.universe/tree/main/sensing/gnss_poser) and [geo_pos_conv](https://github.com/autowarefoundation/autoware.universe/tree/main/sensing/geo_pos_conv)) that seem to do a similar thing with this repository.

Have you ever considered creating PRs to these packages rather than adding new repository?

Reply:
The ealgeyeye is not only used for autoware, but the packages you suggested are in autoware.universe or use their own messages, and it was hard to incorporate them into ealgeye, so we did not modify or work on them this time.
Is it possible, for example, to manage those packages in autoware.universe and a separate repository?

Reply:
If we cannot use the node directly, I would at least would like to see some discussion between Map IV and TIER IV localization team about extraction of library to convert between lat/lon <-> x,y,z coordinates.
At least I see significant overlap in:
* https://github.com/MapIV/llh_converter/blob/main/include/llh_converter/llh_converter.hpp
* https://github.com/autowarefoundation/autoware.universe/blob/main/sensing/gnss_poser/include/gnss_poser/convert.hpp
so that we are using same projection between different nodes.

Reply:
OK, I would like to discuss it once on the MAP IV side and then bring the discussion to TIER IV localization team.

Reply:
I have discussed with @rsasaki0109 and @YamatoAndo on this.
In conclusion, we can merge this PR as a separate package for now to avoid unnecessary dependency between Autoware and Eagleye. However, as a long term, we would like to extract it as a common library once code is more stable.

Reply:
https://github.com/autowarefoundation/autoware.universe/issues/2915